purchases/src/test/java/com/revenuecat/purchases/identity/IdentityManagerTests.kt

```diff
@@ -522,16 +522,13 @@ class IdentityManagerTests {
             true
         )
         identityManager.configure(userId)
-        verify(exactly = 1) {
-            mockDeviceCache.clearCustomerInfoCache(userId, mockEditor)
-        }
```

Comment:
Perhaps the same as in iOS should be done here (verify that `clearCustomerInfoCache` is not invoked)?